<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.1.1 (Python 3.7.2 on win32)" generated="20190326 11:12:02.448" rpa="false">
<suite id="s1" name="Amazon" source="D:\Git\github\Robot_Framework_Recap\Robot_Framework_Recap\Tests\amazon.robot">
<test id="s1-t1" name="User must sign in to check out">
<kw name="Open Browser" library="SeleniumLibrary">
<doc>Opens a new browser instance to the given ``url``.</doc>
<arguments>
<arg>http://www.amazon.com</arg>
<arg>gc</arg>
</arguments>
<msg timestamp="20190326 11:12:02.586" level="INFO">Opening browser 'gc' to base url 'http://www.amazon.com'.</msg>
<kw name="Capture Page Screenshot" library="SeleniumLibrary">
<doc>Takes screenshot of the current page and embeds it into log file.</doc>
<msg timestamp="20190326 11:12:02.588" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<status status="PASS" starttime="20190326 11:12:02.588" endtime="20190326 11:12:02.588"></status>
</kw>
<msg timestamp="20190326 11:12:02.590" level="FAIL">WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
</msg>
<status status="FAIL" starttime="20190326 11:12:02.586" endtime="20190326 11:12:02.590"></status>
</kw>
<doc>This is some basic info</doc>
<tags>
<tag>Smoke</tag>
</tags>
<status status="FAIL" starttime="20190326 11:12:02.585" endtime="20190326 11:12:02.590" critical="yes">WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home
</status>
</test>
<status status="FAIL" starttime="20190326 11:12:02.449" endtime="20190326 11:12:02.591"></status>
</suite>
<statistics>
<total>
<stat pass="0" fail="1">Critical Tests</stat>
<stat pass="0" fail="1">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="1">Smoke</stat>
</tag>
<suite>
<stat pass="0" fail="1" id="s1" name="Amazon">Amazon</stat>
</suite>
</statistics>
<errors>
</errors>
</robot>
